### ISSUE TITLE
Missing 'position' property in Api Gateway client 'getUsage' result

### ISSUE THREAD

**Issue by costasovo** [aws/aws-sdk-php#1486]
I'm using `Aws\ApiGateway\ApiGatewayClient` method `getUsage`.  
My SDK version is 3.52.2.

According to [documentation](https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-apigateway-2015-07-09.html#getusage) the result should contain `position` property which should be used for paging, but the property is missing. Is it a bug or feature? :)

Since there is a region limit for API keys count I made a workaround using the high limit number.

```
$gatewayClient->getUsage([
	'startDate' => $dateFrom->format('Y-m-d'),
	'endDate' => $dateTo->format('Y-m-d'),
	'usagePlanId' => $usagePlanId,
	'limit' => $highEnoughLimit,
]);
```

I would  prefer to use the paging since I can't be sure that the $highEnoughLimit will be high enough forever...

**Comment by kstich** [comment]
You can use [the built in `getPaginator` functionality, specifying the `'GetUsage'` API call](https://docs.aws.amazon.com/aws-sdk-php/v3/guide/guide/paginators.html) and your parameters to handle pagination automatically. Many times, a paginated operation will not return the token (in this case `'position'`) if the response has not been paginated.

**Comment by costasovo** [comment]
Thank you I didn't know about paginators in the SDK.  I tried to use it but I don't get the results for all api keys :(

```
$results = $gatewayClient->getPaginator('GetUsage', [
	'startDate' => $dateFrom->format('Y-m-d'),
	'endDate' => $dateTo->format('Y-m-d'),
	'usagePlanId' => $usagePlan,
]);

foreach ($results as $result) {
	$items = array_merge($items, $result['items']);
}
```

I thought ( from the article )that while I iterate the paginator will make subsequent api calls and I will get the data for all keys in the usage plan.  In this example the foreach body was executed only once and I got 20 results but I know that there should be 90 results for given input.

I tried also the async aproach `$results->each(...)` with same result.

When I dump the `$result['items']` there is still not the `position` property.

Maybe I'm still missing something?
